### DISCUSSION TITLE
Tell me about Google Play Pass

### DISCUSSION THREAD

**Discussion by IShix-g** [KirillKuzyk/Google-Play-License-Check#5] [Q&A] [answered]
If `LicenseStatus` is `LICENSED`, I recognize that Google Play Pass is enabled. Am I right in my thinking? My game is a free game.

```C#
GooglePlayLicense.Check(license =>
{
    if (license.status == LicenseStatus.LICENSED)
    {
        // Google Play Pass is enabled.
    }
    else
    {
        // Google Play Pass is disabled. or There's some kind of error.
    }
});
```

**Comment by KyryloKuzyk** [comment, marked as answer]
If the status code is LICENSED, it can mean that:
1. A user downloaded a free game on Google Play.
2. A user has purchased a paid game.
3. A user has an active Google Play Pass subscription.

In all cases, the LICENSED code indicates that the app was downloaded from Google Play (wasn't sideloaded and wasn't tempered with).

**Reply by IShix-g** [reply]
Thank you for your precise answer.
